inference-engine/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -26,6 +26,24 @@
 namespace ov {
 namespace test {
 
+std::ostream& operator <<(std::ostream& os, const std::vector<ov::Shape>& shapes) {
+    os << "{";
+    for (auto i = 0; i < shapes.size(); ++i) {
+        const auto& shape = shapes[i];
+        os << shape;
+        if (i < (shapes.size() - 1ul)) {
+            os << ", ";
+        }
+    }
+    os << "}";
+    return os;
+}
+
+std::ostream& operator <<(std::ostream& os, const std::pair<ov::PartialShape, std::vector<ov::Shape>>& inputShape) {
+    os << inputShape.first << ": " << inputShape.second;
+    return os;
```

Comment:
Again, the `<<` operator use comma sings as a separator. And, as I have mentioned before, we already use some kind of template on how we dump shapes definition please see Eltwise and Softmax common layer test classes for the reference.

The algorithm is something like that. So it seems to me that it is better to use consistent input shapes description in test names.

```cpp
    result << "IS=" << CommonTestUtils::partialShape2str({shapes.first}) << "_";
    result << "TS=";
    for (const auto& item : shapes.second) {
        result << CommonTestUtils::vec2str(item) << "_";
    }
```

Reply:
done, in `ov_subgraph.cpp`